README_dev.md

```diff
@@ -0,0 +1,208 @@
+variable is empty**. Ensure that it remains empty when running Cubit as well.
+
+```bash
+unset LD_LIBRARY_PATH
+```
+
+Build MOAB
+==========
+
+MOAB must be built with HDF5 enabled. On Ubuntu 18.04, HDF5 is located in the
+`/usr/lib/x86_64-linux-gnu/hdf5/serial` directory, but it may be located
+somewhere else on other flavors or versions of Linux. MOAB should be built with
+the Eigen matrix algebra library instead of LAPACK. The
+`_GLIBCXX_USE_CXX11_ABI=0` flag is required for compatibility with Cubit.
+
+The following commands show how to correctly build the MOAB dependency. If HDF5
+is located somewhere other than `/usr/lib/x86_64-linux-gnu/hdf5/serial`, then
+replace the directory with the correct one.
+
+```bash
+cd ${HOME}/plugin-build
+mkdir -pv moab/bld
+cd moab
+git clone https://bitbucket.org/fathomteam/moab -b 5.3.0
+cd moab
+# patching MOAB CMakeLists.txt to use default find(HDF5)
+$SED -i "s/HDF5_MOAB/HDF5/" CMakeLists.txt
+cd ..
+#end of patch
+cd bld
+cmake ../moab -DENABLE_HDF5=ON \
+        -DHDF5_ROOT=$HDF5_PATH \
+        -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS} \
+        -DENABLE_BLASLAPACK=OFF \
+        -DENABLE_FORTRAN=OFF \
+        $CMAKE_ADDITIONAL_FLAGS \
+        -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/moab
+make
+make install
+cd ../..
+rm -rf moab/moab moab/bld
+```
+
+Build DAGMC
+===========
+
+The following commands show how to build the DAGMC dependency. The `uwuw` and
+`make_watertight` features should be turned on, while other features should be
+turned off. The `MOAB_DIR` variable should point to the location of the
+previously-built MOAB library. The `_GLIBCXX_USE_CXX11_ABI=0` flag is once again
+required.
+
+The following commands show how to correctly build the DAGMC dependency.
+
+```bash
+cd ${HOME}/plugin-build
+mkdir -pv DAGMC/bld
+cd DAGMC
+git clone https://github.com/svalinn/DAGMC -b develop
+cd bld
+cmake ../DAGMC -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0 \
+               -DMOAB_DIR=${HOME}/plugin-build/moab \
+               -DBUILD_UWUW=ON \
+               -DBUILD_TALLY=OFF \
+               -DBUILD_BUILD_OBB=OFF \
+               -DBUILD_MAKE_WATERTIGHT=ON \
+               -DBUILD_SHARED_LIBS=ON \
+               -DBUILD_STATIC_LIBS=OFF \
+               -DCMAKE_BUILD_TYPE=Release \
+               -DCMAKE_INSTALL_PREFIX=${HOME}/plugin-build/DAGMC
+make -j`grep -c processor /proc/cpuinfo`
+make install
+```
+
+Build the Plugin
+================
+
+The following commands show how to build the plugin itself. The `CUBIT_ROOT`
+variable should point to the location of Cubit. The `DAGMC_DIR` variable should
+point to the location of the previously-built DAGMC library.
+
+```bash
+cd ${HOME}/plugin-build
+mkdir -pv bld
+cd bld
+cmake ../Cubit-plugin -DCUBIT_ROOT=PATH_TO_CUBIT \
+                       -DDAGMC_DIR=${HOME}/plugin-build/DAGMC \
+                       -DCMAKE_BUILD_TYPE=Release \
+                       -DCMAKE_INSTALL_PREFIX=${HOME}/plugin-build
```

Comment:
```suggestion
cmake ../Cubit-plugin -DCUBIT_ROOT=PATH_TO_CUBIT \
                       -DDAGMC_DIR=${HOME}/plugin-build/DAGMC \
                       -DCMAKE_BUILD_TYPE=Release \
                       -DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0 \
                       -DCMAKE_INSTALL_PREFIX=${HOME}/plugin-build
```

Reply:
`-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0` is required on the plugin level as well